common.sh

```diff
@@ -464,7 +465,7 @@ solution_file() {
     # solution file will be created in module directory
     # :param message: Message - string of list of strings
     #
-    echo "$1" >> "$SOLUTION_FILE"
+    echo "$@" >> "$SOLUTION_FILE"
```

Comment:
This works. And what about the other functions, like exit_failed, ...? I guess, it should implement the same functionality.